bundles/net.tourbook.device/src/net/tourbook/device/suunto/SuuntoJsonProcessor.java

```diff
@@ -153,10 +152,10 @@ public int compare(final TimeData firstTimeData, final TimeData secondTimeData)
       }
 
       // If the activity contains laps, we need to close the last lap.
-      if (numLaps != 0) {
+      if (_lapCounter != 0) {
```

Comment:
Since a while, I'm using the prefix `num` for `number of` to avoid longer names, e.g. `numberOf` or `counter` -> `numLaps`

Reply:
Fixed.